pkg/agent/agent_test.go

```diff
@@ -151,3 +151,21 @@ func TestPersistRoundNum(t *testing.T) {
 	// expectations above).
 	persistRoundNum(roundNum, mockOVSBridgeClient, 0, maxRetries)
 }
+
+func TestGetRoundInfo(t *testing.T) {
+	controller := mock.NewController(t)
+	defer controller.Finish()
+	mockOVSBridgeClient := ovsconfigtest.NewMockOVSBridgeClient(controller)
+
+	mockOVSBridgeClient.EXPECT().GetExternalIDs().Return(nil, ovsconfig.NewTransactionError(fmt.Errorf("Failed to get external IDs"), true))
+	roundInfo := getRoundInfo(mockOVSBridgeClient)
+	if roundInfo.RoundNum != 1 {
```

Comment:
I would prefer the more concise:
```
assert.Equal(t, 1, roundInfo.RoundNum, "Unexpected round number")
```

Since you are in the same package you do not need to hardcode `1` as well. You can use `initialRoundNum`.